osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -64,29 +65,28 @@ public partial class ChannelManager : CompositeComponent, IChannelPostTarget
         public IBindableList<Channel> AvailableChannels => availableChannels;
 
         private readonly IAPIProvider api;
-        private readonly IChatClient chatClient;
 
         [Resolved]
         private UserLookupCache users { get; set; }
 
         private readonly IBindable<APIState> apiState = new Bindable<APIState>();
         private ScheduledDelegate scheduledAck;
 
+        private IChatClient chatClient = null!;
         private long? lastSilenceMessageId;
         private uint? lastSilenceId;
 
         public ChannelManager(IAPIProvider api)
         {
             this.api = api;
 
-            chatClient = api.GetChatClient();
```

Comment:
Please read description of commit https://github.com/ppy/osu/pull/34064/commits/0110a568de299122c4871bc0115fec6f1f885506 to understand what's going on here.